src/App.tsx

```diff
@@ -8,7 +8,105 @@ import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { getUser } from './api';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { TodoInfo } from './types/TodoInfo';
+
 export const App: React.FC = () => {
+  const [selectedTodoId, setSelectedTodoId] = useState<number | null>(null);
+  const [isLoad, setIsLoad] = useState(false);
```

Comment:
Consider renaming `isLoad` to `isLoading` for better readability and to follow common naming conventions for boolean variables.